batchread/batchread_test.go

```diff
@@ -131,12 +129,9 @@ func collectStats(averageDurationCalculator *averageDurationType, duration time.
 	averageReadDurationStr := fmt.Sprintf("  %v  ", averageDurationCalculator.getAverage())
 	maxDurationStr := fmt.Sprintf("  %v  ", averageDurationCalculator.getMaxDuration())
 	minDurationStr := fmt.Sprintf("  %v  ", averageDurationCalculator.getMinDuration())
-
 	line := []string{runTypeStr, batchSizeStr, durationStr, averageReadDurationStr, maxDurationStr, minDurationStr, readNumStr}
 	err = w.Write(line)
 	panicIfError(err)
-	w.Flush()
```

Comment:
If you do not flush the CSV, good chances it will not be written

Reply:
line 117 - defer flush

Reply:
What is the order of defer execution? the flush has to happen before the close

Reply:
Found out it is LIFO - no problem